FirebaseAuthExchange/Sources/Models/ExchangeServiceModels.swift

```diff
@@ -60,3 +60,66 @@ struct ExchangeOIDCTokenRequest: Encodable {
     self.authCodeCredentials = authCodeCredentials
   }
 }
+
+// MARK: - Response Models
+
+/// Response that encapsulates an Auth Exchange access token.
+///
+/// This is the return value for an `ExchangeInstallationAuthTokenRequest` or
+/// `ExchangeCustomTokenRequest`.
+struct ExchangeTokenResponse: Decodable, Equatable {
+  /// A container for a Firebase  access token and the duration of time until it expires.
+  struct AuthExchangeToken: Decodable, Equatable {
+    /// A signed [JWT](https://tools.ietf.org/html/rfc7519) containing claims that identify a user.
+    let accessToken: String
+
+    /// The duration of time until the `accessToken` expires, approximately relative to the time
+    /// this response was received.
+    let timeToLive: ProtobufDuration
+
+    enum CodingKeys: String, CodingKey {
+      case accessToken
+      case timeToLive = "ttl"
```

Comment:
Yeah I agree, `timeToLive` makes more sense from our end as well. Gonna go ahead and add a similar notion in ours to do the same (even though it won't go beyond network calls- still looks cleaner)

:)

Reply:
Thanks! That'll make it easier for anyone reviewing both codebases.